util/container/README.md

```diff
@@ -7,8 +7,11 @@ software development tools for OpenTitan. Current list of tools:
 * fusesoc
 * OpenOCD
 * RISCV toolchain
+* clang-format
```

Comment:
Should this be `clang-format-3.8` (to match the actual deb pkg name)? Plus, this explicitly indicates what version we are talking about.

Reply:
I didn't think the exact version was important - I thought the important part to note was `clang-format` was available, and that the version matched the CI.

Reply:
Okay, fine with me either way.